src/sql/workbench/parts/notebook/common/models/notebookInput.ts

```diff
@@ -0,0 +1,463 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
This file is now in 2 places, what's going on?

Reply:
Fixed, thanks